Cargo.toml

```diff
@@ -10,14 +10,22 @@ homepage = "https://github.com/pingcap/raft-rs"
 documentation = "https://docs.rs/raft"
 description = "The rust language implementation of Raft algorithm."
 categories = ["algorithms", "database-implementations"]
+build = "build.rs"
+
+[build-dependencies]
+protoc-rust = "2.2.4"
+protoc-grpcio = "0.3.1"
+regex = "1.1"
 
 [features]
-default = []
+default = ["lib-rust-protobuf"]
+lib-rust-protobuf = ["protobuf"]
+regenerate = []
```

Comment:
This needs some documentation in `lib.rs` please. :)

Please make sure the user knows how to accomplish the main use cases this provides. Eg regenerating the protobufs.

Reply:
Actually I'm having trouble on using features, the `BufferLib::from_env_vars()` in build.rs does not seem to return `BufferLib::Prost` when I build this with `cargo build --features lib-prost`.

Reply:
FYI the `lib-prost = [...]` is removed because we don't introduce any prost things in this PR.

Reply:
> This needs some documentation in `lib.rs` please. :)

How about `README.md` 😉?

Reply:
Both might be appropriate, but README.md is not part of generated docs.rs documentation. :(

Reply:
I'll update README in prost PR. I don't think it's proper to introduce not-completely-ready features in README.

Reply:
OK.